parallel-jvm/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ./namespace-0
+- ./namespace-1
```

Comment:
It might be more concise skip the per-namespace kustomizations, e.g.:

`namespace.yaml`:
```
apiVersion: v1
kind: Namespace
metadata:
  name: jvm-exp-0
  labels:
    experiment: jvm-reactors
---
apiVersion: v1
kind: Namespace
metadata:
  name: jvm-exp-1
  labels:
    experiment: jvm-reactors
```

Technically, nothing prevents you from sticking everything in the `experiment.yaml` if you just want to be extra lean.

Reply:
Yeah good idea i went with the lean version